scripts/e2e-test.sh

```diff
@@ -12,7 +12,8 @@ TAG=
 TESTDIR=$(realpath "$SCRIPTDIR/../test/e2e")
 REPORTSDIR=$(realpath "$SCRIPTDIR/..")
 GENERATE_LOGS=0
-ON_FAIL="continue"
+ON_FAIL="stop"
+CLEANUP="0"
```

Comment:
Please use lowercase for shell variables (compared to constants).

Reply:
Done